src/routes/docs/products/functions/develop-locally/+page.markdoc

```diff
@@ -71,6 +75,81 @@ You can use headers like dynamic API keys in your function, which give you acces
 Learn more about dynamic API keys
 {% /arrow_link %}
 
+# Hot reloading {% #hot-reloading %}
```

Comment:
This can just be "Hot reload" instead of "Hot reloading".

In addition, if we're going to add a section, we can remove the sentences mentioning this feature at the bottom of the Develop section.

Reply:
I'm not sure I see the sentences mentioning hot reloading in the Develop docs or the Local Develop docs. What are you referring to remove here?